XSLT-to-SVG-1/mollyXSLT/blobs-to-SVG.xsl

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math" xmlns="http://www.w3.org/2000/svg"
+    exclude-result-prefixes="xs math" version="3.0">
+    <!-- ebb: This XSLT is designed to output SVG from information in an XML file.
+        We need the SVG namespace to control the output in the XSLT root element:
+        see the xmlns="http://www.w3.org/2000/svg" 
+        
+        The XSLT is reading in XML that has no namespace. 
+        (If you are reading from XML in a namespace such as TEI,
+        use the xpath-default-namespace="..." to apply it.) 
+    -->
+
+    <xsl:variable name="x-spacer" select="200"/>
+
+    <xsl:variable name="y-spacer" select="10"/>
+
+    <xsl:output method="xml" indent="yes"/>
+
+
+    <xsl:template match="/">
+        <svg width="100%">
+            <desc>SVG created from <xsl:apply-templates select="//title"/></desc>
+
+            <!-- I'm applying transform="translate()" to anticipate that I need a plot with y values in the 500s.
+                0,0 will move down the screen to 20, 500. 
+            See https://www.w3schools.com/graphics/svg_transformations.asp 
+            -->
+            <g transform="translate(20 500)">
+
+                <xsl:for-each select="descendant::blob">
+
+                    <!-- We'll process the blob elements in here and output a shape 
+                    for each one -->
+                    <circle cx="{$x-spacer * position()}" cy="{100}"
+                        r="{count(descendant::special[@whatsIt='magicEgg']) * $y-spacer}"
+                        fill="red" stroke="black"/>
+                    <text x="{$x-spacer * position() - 45}" y="100" fill="none" stroke="black" font-size="20">Magic Egg</text>
+                </xsl:for-each>
+
+                <xsl:for-each select="descendant::blob">
```

Comment:
@mo-wright This is amazing work for your first ever XSLT! Your output is showing on the screen so that's good. But you don't need all of the `<xsl:for-each>` elements here. You only need one of them! Try this one revision: just keep **one** `<xsl:for-each select="descendant:blob">` : That one is going into the source XML file and finding each `<blob>` element. For each of these, you're asking it output as many shapes as  you wish in relation to each other! 

So you can tuck all the shape and text elements inside the same `<xsl:for-each>`.